src/main/java/com/webonise/controllers/LatLngController.java

```diff
@@ -0,0 +1,48 @@
+package com.webonise.controllers;
+
+import com.webonise.models.LatLng;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LatLngController {
+
+    private ObservableList<LatLng> latLngList;
```

Comment:
If you are initializing this list inside constructor then why you didnt make it thread safe?